src/substrate-service.ts

```diff
@@ -66,6 +66,32 @@ export class SubstrateService {
     });
   }
 
+  public async issueAssetToUserBatch(amount: string, wallets: Array<string>): Promise<IssueAssetResponse> {
+    let i
+    let txs = []
```

Comment:
Let's put a semicolon at the end of the lines as well.

Reply:
@nikohosn 
Be sure you've installed prettier correctly in you IDE.